fala/assets-src/sass/main.scss

```diff
@@ -187,3 +187,11 @@ body.fala-dev {
   margin-top:1em;
   clear:both;
 }
+
+.miles-away {
+  color: $govuk-secondary-text-colour;
+}
+
+.results-list-item {
```

Comment:
**Optional**, I'm not going to lose sleep over this because it's easy to refactor. 

I would follow BEM here for future proofing: https://getbem.com/

So you can do:

```
.results-list
&__item {
margin-top: govuk-spacing(4);
}
```

This allows extensions and future styling for the results list items or even something else on the result page.